rubycritic.gemspec

```diff
@@ -46,6 +51,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '>= 2.0.0'
   if RUBY_PLATFORM == 'java'
     spec.add_development_dependency 'pry-debugger-jruby'
+    spec.add_development_dependency 'jar-dependencies', '~> 0.4.1'
```

Comment:
Given the comment about jar-dependencies in https://www.jruby.org/2025/02/11/jruby-9-4-12-0.html, "jar-dependencies upgraded to 0.5.4 to fix an issue parsing Maven output on Java versions 9 and higher," should we lock jar-dependencies to `-> 0.5.4`, or even remove the version restriction so as to pick up whatever jruby wants to install?